.npmrc

```diff
@@ -0,0 +1,2 @@
+@adobe:registry=https://registry.npmjs.org/
```

Comment:
What do we get from adding these?

Reply:
Now I see what this is related to. That's been in the README.md forever, but I'm not sure I've ever done it... Listing here makes sense, and in doing so does that means we can remove it from the README?